docs/development_environment.md

```diff
@@ -46,130 +44,54 @@ It is also possible to set up a more traditional development environment. See th
-
-```shell
-Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
-```
-
-From Windows Store install Ubuntu.
-
-#### For those already running Ubuntu 18.04 on WSL
-
-If you are running Ubuntu 18.04 on Windows Subsystem for Linux some additional steps are needed to install the latest FFmpeg version.
-
-```shell
-$ sudo add-apt-repository ppa:savoury1/ffmpeg4
-$ sudo add-apt-repository ppa:savoury1/graphics
-$ sudo add-apt-repository ppa:savoury1/multimedia
-$ sudo apt-get update
-$ sudo apt-get install ffmpeg
-```
-
-When the Linux subsystem is set up, perform install as for Linux.
+To develop on Windows, you will need to use the Linux subsystem. Follow the [WSL installation instructions](https://docs.microsoft.com/windows/wsl/install-win10) and install Ubuntu from the Windows Store. Once you're able to access Linux, follow the Linux instructions.
 
-```shell
-$ sudo apt-get update
-$ sudo apt-get install python3-pip python3-dev python3-venv python-wheel-common
-$ sudo apt-get install autoconf libssl-dev libxml2-dev libxslt1-dev libjpeg-dev libffi-dev libudev-dev zlib1g-dev
-$ sudo apt-get install -y libavformat-dev libavcodec-dev libavdevice-dev libavutil-dev libswscale-dev libavresample-dev libavfilter-dev
-```
-
-Hint: Git is included in Linux subsytem.
-
-When invoking your installation (see below), make sure to specify a folder for configuration which is accessible from Windows.
-
-```shell
-$ mkdir -p ../config
-$ hass -c ../config
-```
-
-:::tip
-You may find that you cannot open the development instance via <http://localhost:8123> when using WSL. Instead, within a WSL terminal, find the `inet` address of the `eth0` adaptor by running `ip addr show eth0`. Then use this address, excluding the CIDR block, to access the development instance, i.e. if your `inet` is listed as `172.20.37.6/20`, use <http://172.20.37.6:8123>.
-:::
+If you find that you cannot open the development instance via <http://localhost:8123> when using WSL. Instead, within a WSL terminal, find the `inet` address of the `eth0` adaptor by running `ip addr show eth0`. Then use this address, excluding the CIDR block, to access the development instance, i.e. if your `inet` is listed as `172.20.37.6/20`, use <http://172.20.37.6:8123>.
 
 ### Developing on macOS
 
-Install [Homebrew](https://brew.sh/), then use that to install Python 3:
-
-```shell
-$ brew install python3 autoconf
-```
-
-Then install ffmpeg:
+Install [Homebrew](https://brew.sh/), then use that to install the dependencies:
 
 ```shell
-$ brew install ffmpeg
+$ brew install python3 autoconf ffmpeg
 ```
 
 ## Setup Local Repository
 
 Visit the [Home Assistant Core repository](https://github.com/home-assistant/core) and click **Fork**.
 Once forked, setup your local copy of the source using the commands:
 
-_Windows users should be sure to clone to a path that inside the WSL (ex: ~/)._
+_Windows users should be sure to clone to a path that is inside the WSL (ex: ~/)._
```

Comment:
WSL now mounts all the windows drives so this point probably isn’t needed anymore